doc/nrf/protocols/matter/end_product/factory_provisioning.rst

```diff
@@ -13,6 +13,898 @@ The factory data parameter set includes different types of information, for exam
+
+All parameters of the factory data set are either mandatory or optional:
+
+* Mandatory parameters must always be provided, as they are required for example to perform commissioning to the Matter network.
+* Optional parameters can be used for development and testing purposes.
+  For example, the `user` data parameter consists of all data that is needed by a specific manufacturer and that is not included in the mandatory parameters.
+
+In the factory data set, the following formats are used:
+
+* uint8, uint16, and uint32 - These are the numeric formats representing, respectively, one-byte length unsigned integer, two-bytes length unsigned integer, and four-bytes length unsigned integer.
+  This value is stored in a HEX file in the big-endian order.
+* Byte string - This parameter represents the sequence of integers between `0` and `255`(inclusive), without any encoding.
+  Because the JSON format does not allow to use of byte strings, the `hex:` prefix is added to a parameter, and its representation is converted to a HEX string.
+  For example, an ASCII string _`abba`_ is represented as _`hex:61626261`_ in the JSON file and then stored in the HEX file as `0x61626261`.
+  The HEX string length in the JSON file is two times greater than the byte string plus the size of the prefix.
+* ASCII string is a string representation in ASCII encoding without null-terminating.
+* ISO 8601 format is a `ISO 8601 date format`_ that represents a date provided in the `YYYY-MM-DD` or `YYYYMMDD` format.
+* All certificates stored in factory data are provided in the `X.509`_ format.
+
+.. _ug_matter_device_factory_provisioning_appearance_field_description:
+
+Appearance field description
+----------------------------
+
+The `appearance` field in the factory data set describes the device's visible appearance.
+
+* `finish` - A string name that indicates the visible exterior finish of the product.
+  It refers to the `ProductFinishEnum` enum, and currently, you can choose one of the following names:
+
++------------+------------+
+| Name       | Enum value |
++------------+------------+
+| `matte`    | 0          |
++------------+------------+
+| `satin`    | 1          |
++------------+------------+
+| `polished` | 2          |
++------------+------------+
+| `rugged`   | 3          |
++------------+------------+
+| `fabric`   | 4          |
++------------+------------+
+| `other`    | 255        |
++------------+------------+
+
+* `primary_color` - A string name that represents the RGB color space of the device's case color, which is the most representative.
+  It refers to the `ColorEnum` enum, and currently, you can choose one of the following names:
+
+(Enum value) color name (`RGB value`)
+
+| (0) $$\color{black} \color{black}{black}$$ (`#000000`) | (1) $$\color{#000080}{navy}$$ (`#000080`)    | (2) $$\color{#008000}{green}$$ (`#008000`)    | (3) $$\color{#008080}{teal}$$ (`#008080`)    | (4) $$\color{#800080}{maroon}$$ (`#800080`)             |
```

Comment:
Is this supposed to be a table? 
<img width="730" height="276" alt="image" src="https://github.com/user-attachments/assets/cbf04af9-9f05-4d1b-93fa-48339e8c0401" />